src/stoppingtest.jl

```diff
@@ -17,10 +28,11 @@ function test_stopping_criterion(param::SDDPparameters, stats::SDDPStat)
     lb = stats.lower_bounds[end]
     ub = stats.upper_bounds[end]
     check_gap = (abs((ub-lb)/lb) < param.gap)
-    check_iter = stats.niterations > param.maxItNumber
+    check_iter = stats.niterations >= param.maxItNumber
```

Comment:
Actually when the test is done we have updated niterations. So I'm changing back to strict inequality.